### PR TITLE
SRVLOGIC-261-hotfix: Include workflow plugin binary download install

### PR DESCRIPTION
**Pull request by domhanak**
Add hotfix for https://issues.redhat.com/browse/SRVLOGIC-261

This reintroduces the basic kn workflow plugin install procedure using binary download

### CODE REVIEW

File: modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc [hunk 1]
```diff
@@ -7,7 +7,40 @@
 
 *Prerequisites*
 
-* You have first installed the  link:{kn_cli_install_url}[Knative CLI].
+* You have first installed the link:{kn_cli_install_url}[Knative CLI].
+* link:{java_install_url}[Java] {java_min_version} is installed.
+* link:{maven_install_url}[Maven] {maven_min_version} or later is installed.
```
Comment (ricardozanini) [inline modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc:12]: Are you sure maven and Java are required? The run command will execute everything within the container that has these requirements already.

File: modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc [hunk 2]
```diff
@@ -7,7 +7,40 @@
...
+* (Optional) link:{docker_install_url}[Docker] is installed.
+* (Optional) link:{podman_install_url}[Podman] is installed.
```
Comment (ricardozanini) [inline modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc:14]: Docker and podman are not optional. Either one of them must be installed.

File: modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc (hunk 1 above)
Comment (domhanak) [reply modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc:12]: Wanted to keep it in sync with upstream, but yea will remove it and sent upstream pr too.

File: modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc (hunk 2 above)
Comment (domhanak) [reply modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc:14]: Will fix.

File: modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc (hunk 1 above)
Comment (domhanak) [reply modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc]: add. I think the idea in upstream is that there are quarkus commands being shown in the guide

Comment (ricardozanini) [reply modules/serverless-logic/pages/cloud/operator/install-kn-workflow-cli.adoc]: Sure, that's another thing we are going to remove.

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**